pkg/controller/accountclaim/reuse.go

```diff
@@ -85,6 +85,26 @@ func (r *ReconcileAccountClaim) finalizeAccountClaim(reqLogger logr.Logger, acco
 		if err != nil {
 			reqLogger.Error(err, "Failed to delete BYOC account from accountclaim cleanup")
 		}
+
```

Comment:
Same as the other comment, can we abstract this out to another function and unit test it?